src/components/blog/BlogPostsContent.js

```diff
@@ -58,11 +59,17 @@ const CtaContainer = styled.div`
 
 `;
 
-const BlogPostsContent = ({postsPerPage, offset, posts, setOffset })=> {
+const BlogPostsContent = ({postsPerPage, offset, posts })=> {
+  const { page } = useParams()
+  if (page <= 0 || (posts.length/postsPerPage < page - 1)) navigate('/')
```

Comment:
Page is subtracted by 1 so that routes are as follow:
- /blog
- /blog/posts/2
- /blog/posts/3
Instead of having route /blog/posts/1 be an equivalent to actual second page of posts